src/IRsend.h

```diff
@@ -318,11 +318,17 @@ class IRsend {
   void sendVestelAC(const uint64_t data, const uint16_t nbits = kVestelACBits,
                     const uint16_t repeat = kNoRepeat);
 #endif
+#if SEND_TCL112AC
+  void sendTcl112Ac(const unsigned char data[],
```

Comment:
Same `AC` vs `Ac` here

Reply:
Noted: See earlier comment. i.e. AC is wrong, it should be Ac etc.